addon/package.json

```diff
@@ -24,7 +24,10 @@
   "scripts": {
     "build": "rollup --config",
     "prepublishOnly": "rollup --config",
-    "start": "rollup --config --watch"
+    "start": "rollup --config --watch",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
+    "lint:js": "pnpm -w eslint ./addon",
+    "lint:js:fix": "pnpm -w eslint ./addon --fix"
```

Comment:
Run eslint, from the workspace root, so that it uses the monorepo config defined there.